src/store/actions/user.actions.ts

```diff
@@ -105,3 +106,6 @@ export const logout = (): ThunkAction<Promise<void>, RootState, unknown, Action>
   };
 };
 
+export const onLogin = () => {
```

Comment:
move this to the userService. as this is not an action that can be dispatched. so this should not be in the actions

Reply:
moved